src/api/java/cvc5/Solver.java

```diff
@@ -0,0 +1,48 @@
+package cvc5;
+
+import java.io.IOException;
+
+public class Solver implements IPointer
+{
+  private final long pointer;
+
+  public long getPointer()
+  {
+    return pointer;
+  }
+
+  public Solver()
+  {
+    this.pointer = newSolver();
+  }
+
+  private native long newSolver();
+
+  public void deletePointer()
+  {
+    deletePointer(pointer);
+  }
+
+  private static native void deletePointer(long solverPointer);
+
+  static
+  {
+    Utils.loadLibraries();
+  }
+
+  /**
+   * Set logic.
+   * SMT-LIB: ( set-logic <symbol> )
+   *
+   * @param logic
+   * @throws CVC5ApiException
+   */
+  public void setLogic(String logic) throws CVC5ApiException
+  {
+    setLogic(pointer, logic);
+  }
+
+  private native void setLogic(long solverPointer, String logic) throws CVC5ApiException;
+
+  // endregion
```

Comment:
It looks like `// region ...` is missing. Maybe remove the `// endregion`?

Reply:
remnant of old code . Removed